src/compiler/separate_compiler.nit

```diff
@@ -2122,6 +2268,12 @@ class SeparateRuntimeFunction
 	end
 end
 
+redef class MType
+	# Is the type is tagged?
+	# If false, it means that the type is not promitive, or is boxed.
```

Comment:
I don't know what is a pr**o**mitive type. And at this point, I'm affraid to ask :p